docs/conf.py

```diff
@@ -45,13 +45,13 @@
 
 try:
     import sphinx
-    from pkg_resources import parse_version
 
     cmd_line_template = "sphinx-apidoc -f -o {outputdir} {moduledir}"
     cmd_line = cmd_line_template.format(outputdir=output_dir, moduledir=module_dir)
 
     args = cmd_line.split(" ")
-    if parse_version(sphinx.__version__) >= parse_version("1.7"):
+    if tuple(sphinx.__version__.split(".")) >= ("1", "7"):
```

Comment:
Usually it is better to rely on proper parsing, but since Sphinx don't do something super fancy, I think we can trust this code (also passes on system tests)  and avoid dependencies.

Reply:
Yes, agreed.